src/components/navigation/footer/Footer.tsx

```diff
@@ -22,22 +25,34 @@ export interface IFooter {
   companyInfo: CompanyInfo;
   brandAssets: BrandAssets;
   soMeData: SocialMediaProfiles | null;
-  legalData: LegalDocument[];
 }
 
 const Footer = ({
   navigationData,
   companyInfo,
   brandAssets,
   soMeData,
-  legalData,
 }: IFooter) => {
   const renderedLogo = useConvertSanityImageToNextImage(
     brandAssets?.secondaryLogo,
   );
 
   const currentYear = new Date().getFullYear();
 
+  const { defaultLanguage, language } = useLanguage();
+
+  const [legalData, setLegalData] = useState<LegalDocument[] | null>(null);
+
+  useEffect(() => {
```

Comment:
why can't we fetch server side?

Reply:
This was originally because it did not seem like path params could be accessed in layout.tsx. However, it's seems to be possible after all, so I will try to fetch server side :+1:

Reply:
Updated now ✅